package.json

```
@@ -13,7 +13,7 @@
     "verify": "yarn -s lint && yarn types"
   },
   "dependencies": {
-    "@actual-app/api": "4.1.5",
+    "@actual-app/api": "^4.1.6",
```

Can you pin this dependency back? Want to make sure previous versions don’t accidentally break with future releases of the API

```suggestion
    "@actual-app/api": "4.1.6",
```

---

Makes sense. I'll also slip in `.editorconfig` addition here :)
https://github.com/actualbudget/actual-server/pull/118/commits/f070ddca905bbca93985cfa86cc134e5793ceb0c